### ISSUE TITLE
The ability to fit the mixture of distributions with carefull identifiability analysis is needed

### ISSUE THREAD

**Issue by godzi** [cokelaer/fitter#26]
Need more functions to work with mixture of distributions

**Comment by cokelaer** [comment]
I agree; would be very nice. I had a code do work o gaussian mixture in github.com/biokit but that was the only case. not sure this is an easy problem in the general case. Do you have any ideas / suggestions on that matter ?
